Sources/TootSDK/TootClient/TootClient+Trends.swift

```diff
@@ -33,7 +33,7 @@ extension TootClient {
     ///   - offset: Skip the first n results.
     /// - Returns: Array of ``Post``.
     public func getTrendingPosts(limit: Int? = nil, offset: Int? = nil) async throws -> [Post] {
-        try requireFeature(.trends)
```

Comment:
Maybe I'm not seeing it, why does the feature need to be renamed to support the new flavour?